### ISSUE TITLE
Use English in README

### ISSUE THREAD

**Issue by dandv** [gromver/rx-form#9]
https://blog.codinghorror.com/the-ugly-american-programmer/

**Comment by gromver** [comment]
Yes, that's a good idea. But first I wanted to write v3, which will combine the best solutions of rx-form and json-schema. I don't know, maybe on New Year's holidays I will have time for this...
